packages/dids/tests/tech-preview.spec.ts

```diff
@@ -0,0 +1,68 @@
+import type { DwnServiceEndpoint } from '../src/types.js';
+
+import { expect } from 'chai';
+
+import { DidIonApi } from '../src/did-ion.js';
+
+describe('Tech Preview', function () {
+  describe('generateDwnConfiguration()', () => {
+    it.only('returns keys and services with two DWN URLs', async () => {
```

Comment:
what's with the `it.only`s? are you intentionally trying to exclude other tests from the `package/dids/test` suite? If so, can you use `.skip` on the ones you want to skip instead?

Reply:
Thanks for catching.  Should have been removed.